src/main/java/nl/ordina/jobcrawler/model/Vacancy.java

```diff
@@ -1,22 +1,29 @@
 package nl.ordina.jobcrawler.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
Please add comments into the entity classes to tell us what the structure of the database is in some english.

Reply:
Maybe this is best left for the DTO layer, or the Entity Relationship Diagram? To be discussed in the meeting.

Reply:
done